tests/features/solution/add_solution.feature

```diff
@@ -247,3 +247,35 @@ Feature: "Add solution" visibility options.
     Then I delete the "Climate change tracker" solution
     And I delete the "Climate change tracker" solution
     And I delete the "Angela Crespi" contact information
+
+  Scenario: Check solution downloads counter
+    # Create the dummy data to work with.
+    Given the following collection:
+      | title              | Ocean studies |
+    Given the following solution:
+      | title       | Climate change tracker |
+      | description | Atlantic salmon arrived after the Little Ice Age. |
+      | collection  | Ocean studies                                     |
+      | state       | validated                                         |
+    Given the following distributions:
+      | title        | description | parent | downloads |
+      | Sample distribution | Sample description | Climate change tracker | 10 |
+      | Sample distribution 2 | Sample description 2 | Climate change tracker | 0 |
+    # Checks solutions tiles to we have the correct numbers.
+    Then I visit "/solutions"
+    Then I should see the text "10"
+    When I go to the homepage of the "Climate change tracker" solution
+    # Check the same thing on the homepage of the solutions.
+    Then I should see the text "Downloads: 10"
+    When I delete the "Sample distribution" asset distribution
+    Given the cache has been cleared
+    And I reload the page
+    # Check it again after the first distribution is deleted and cache has been cleared.
+    Then I should not see the text "Downloads: 10"
+    # Go back to the solution tiles page and check downloads are disappeared.
+    Then I visit "/solutions"
+    Then I should not see a "icon--download" element
```

Comment:
After this I would love to see adding a distribution with downloads > 0 and check again.

Reply:
What do you mean exactly? Add it like in the "Add a distribution to a solution as a facilitator." scenario or with "Given the following distribution..." and do a check then?

Reply:
> Given the following distribution ...

This should work, I guess.

Reply:
I've commited some changes but it's still not perfect.
When I run this scenario getting an error: **Tile 'Test solution' not found.**
I assume that it will be a caching issue again, because if I clear the cache before I visit /solutions page then it's fine but not a valid use case as you described above somewhere.
Any ideas? Did I do something wrong?

Reply:
This step definition is not understandable for our business stakeholders:

```
Then I should not see a "icon--download" element
```

Our Behat scenarios are intended so our business stakeholders and functional contractors can understand and validate the work that we have done. Not all these people are highly technically skilled and we cannot assume that they will understand what an `icon--download element` is and that they will known how to look for this in the DOM. We need to rewrite this in non-technical language so this will not be rejected in UAT. I would suggest to use an easy to understand step such as:

```
Then the download icon should not be shown in the "Climate change tracker" tile
```

Here is how a step like this could be implemented:

```
/**
 * @Then the download icon should not be shown in the :heading tile
 */
public function assertTileNotContainsDownloadIcon(string $heading): void {
  $tile = $this->getTileByHeading($heading);
  $elements = $tile->findAll('xpath', '//span[contains(@class, "icon--download")]');
  Assert::assertEmpty($elements);
}
```

Reply:
Implemented!